examples/data/SAE_J2411_SWCAN_CPSS.odx-cs

```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- edited with XMLSPY v2004 rel. 2 U (http://www.xmlspy.com) by Christian Hecker (Softing AG) -->

```

Comment:
what's this file?

Reply:
this is a definition of communication parameters which is included in the ZIP file of the ASAM standard and which has been missing so far. The reason why this is now required is that the `CPS_ISO_15765_3_on_ISO_15765_2` comparam spec is now [internalized](https://github.com/mercedes-benz/odxtools/pull/326/files#diff-659a733f80deab14348f5c5c4b5ff936732c06257d5c6d8decf86518eb50735fR2397-R2403) by  `somersaultecu.py` and it features a reference to the `SAE_J2411_SWCAN` comparam subset which is contained in the missing file. I should factor these changes out into the separate "drive bys PR" but it is hard to catch such things when having stared at the code for more than a week. Thanks for the catch!